pkg/log/pipeline_reader.go

```diff
@@ -234,6 +238,13 @@ func hasPipelineRunFailed(prConditions duckv1beta1.Conditions) error {
 	return nil
 }
 
+func isPipelineRunRunning(prConditions duckv1beta1.Conditions) error {
+	if len(prConditions) != 0 && prConditions[0].Status == corev1.ConditionUnknown {
+		return fmt.Errorf("pipelinerun is still running: %s", prConditions[0].Message)
```

Comment:
Please capitalize `PipelineRun` to help with efforts in #605. While it may be odd initially having it lowercase some places, it will eventually all follow this format.

Reply:
addressed